### PR TITLE
Completed javascript conditionals

### PR DESCRIPTION
**Pull request by Andrew-Roide**
javascript-conditionals1
javascript-conditionals2

### CODE REVIEW

File: javascript-conditionals/main.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'The pH is neutral';
+  } else if (pH >= 0 && pH <= 6) {
```
Comment (kurtyazdizadeh) [inline javascript-conditionals/main.js]: What if the pH is 6.5 (or any other number between 6 and 7)?

File: javascript-conditionals/main.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    return 'The pH is an acid';
+  } else if (pH >= 8 && pH <= 14) {
```
Comment (kurtyazdizadeh) [inline javascript-conditionals/main.js]: Same Idea here, what if the pH is 7.5?

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
Make sure to account for the edge cases in the comments I left you.

File: javascript-conditionals/main.js (hunk 1 above)
Comment (Andrew-Roide) [reply javascript-conditionals/main.js]: Looks like I need to brush up on my chemistry as well. This was under the assumption pH levels are only whole numbers!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**